lib/OctokitAction.ts

```diff
@@ -4,8 +4,9 @@ import { GitHub } from '@actions/github/lib/utils';
 import { Action } from './Action';
 import { RestEndpointMethods } from '@octokit/plugin-rest-endpoint-methods/dist-types/generated/method-types';
 import { IssueOrPR } from './IssueOrPR';
-import { Issue, ProjectCard, PullRequest } from './OctokitTypes';
+import { Issue, PullRequest } from './OctokitTypes';
 import { ProjectContent } from './ProjectContent';
+import fetch from 'node-fetch';
```

Comment:
I don't like how this looks, but I haven't found a better way

Reply:
you're following the examples from the doc, so it looks fine to me: https://www.npmjs.com/package/node-fetch